packages/frontend-design-poc/src/pages/SavedSearches/SavedSearchesItem.tsx

```diff
@@ -0,0 +1,61 @@
+import styles from './savedSearches.module.css';
+import { compressQueryParams } from '../Inbox/Inbox';
+import { SavedSearch } from '.';
+import { ChevronRightIcon, EllipsisHorizontalIcon, TrashIcon } from '@heroicons/react/24/outline';
+
+interface SavedSearchesItemProps {
+  savedSearch?: SavedSearch;
+  onDelete?: (id: number) => void;
+}
+
+const RenderButtons = ({ savedSearch, onDelete }: SavedSearchesItemProps) => {
+  if (!savedSearch) return null;
```

Comment:
{}

Reply:
For the record: Jeg personlig liker ikke å legge til 2 ekstra linjer for ingenting. Men legger det inn.